kubernetes/config/samples/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM torchelastic/examples:0.1.0rc1
```

Comment:
Just to confirm - setting up the container this way makes it so you don't have to download the data into the pods like you were doing in the original implementation correct?

Reply:
Also probably explains why you need a different Docker container and cannot use the torchelastic/examples one..

Reply:
We can definitely do that, I can change to original way. During my test, I want to save some time on setting up EFS. That's why I build an image directly